content/docs/tech-insights/builtin-data-sources/index.md

```diff
@@ -10,21 +10,219 @@ description: Built-in Data Sources.
 
 Built-in Data Sources use integration configurations from common Backstage plugins and annotations provided by them. You can see configuration instructions for integrations on their respective documentation pages in [the Plugins & Integrations page](/docs/integrations/).
 
-**Annotations used by built-in data sources**
+You will find a list of built-in Data Sources, below, with annotations used by them and facts they currently include.
```

Comment:
There's a good chance we're going to add more facts to these data sources in future and forget to update these docs. 

Can we explain that there is also an in-app way to get the list of facts available on each data source, and that the in-app list is more likely to be accurate.